CHANGELOG.md

```diff
@@ -0,0 +1,199 @@
+    The minimum supported version remains at `2.8.1` as updated in YoastCS 0.4.1.
+* Composer: Included version of [PHP Mess Detector] has been updated from `2.4.3` to `2.6.0`.
+    The minimum supported version remains at `2.2.3`.
+* PHPCS: YoastCS is now based on the full `WordPress` ruleset, rather than the `WordPress-VIP` ruleset. See: [#16](https://github.com/Yoast/yoastcs/issues/16).
+
+#### Removed
+* PHPCS: The exclusion of the whole `WordPress.NamingConventions.ValidVariableName` sniff.
+    This sniff was previously excluded because of a bug in WordPressCS. This bug has been fixed in WordPressCS 0.10.0.
+
+
+### [0.4.1] - 2017-03-21
+
+#### Added
+* PHPCS: Unit tests for the `Yoast.ControlStructures.IfElseDeclaration` sniff.
+* Travis build testing.
+
+#### Changed
+* Composer: Supported version of [PHP_CodeSniffer] has been changed from `2.7.0` to `~2.8.1`.
+* PHPCS: Exclude function groups rather than individual error codes for WordPressCS sniffs which allow for this.
+
+#### Removed
+* PHPCS: The exclusion for the `Squiz.Commenting.FunctionComment.ScalarTypeHintMissing` error code.
+    This is already taken care of upstream in WordPressCS 0.10.0.
+* PHPCS: Explicit inclusion of the YoastCS native `Yoast.ControlStructures.IfElseDeclaration` sniff from the YoastCS ruleset.
+    All `Yoast` sniffs are automatically included, so the explicit inclusion was redundant.
+
+
+### [0.4] - 2016-09-06
+
+#### Changed
+* Composer: Supported version of [PHP_CodeSniffer] has been changed from `2.5.1` to `~2.7.0`.
+* Composer: Supported version of [WordPressCS] has been changed from `0.9` to `~0.10.0`.
+* Composer: Included version of [PHP Mess Detector] has been updated from `2.4.2` to `2.4.3`.
+    The minimum supported version remains at `2.2.3`.
+
+### [0.3] - 2016-05-03
+
+#### Added
+* PHPCS: An exclusion for the `WordPress.VIP.RestrictedFunctions.get_pages` error code to the YoastCS ruleset.
+
+
+### [0.2] - 2016-04-01
+
+#### Added
+* PHPCS: An exclusion for the `PEAR.Functions.FunctionCallSignature.Indent` error code to the YoastCS ruleset.
+* PHPCS: An exclusion for the `Squiz.Commenting.FunctionComment.ScalarTypeHintMissing` error code.
+* PHPCS: An exclusion for the whole `WordPress.NamingConventions.ValidVariableName` sniff.
+* PHPCS: An exclusion for the `WordPress.VIP.RestrictedFunctions.count_user_posts` error code.
+* PHPMD: `wp` to the list of common exceptions to the `ShortVariable` name rule.
+
+#### Changed
+* Composer: Supported version of [PHP_CodeSniffer] has been changed from `^2.2.0` to `2.5.1`.
+* Composer: Supported version of [WordPressCS] has been changed from `0.6` to `0.9`.
+* Composer: Included version of [PHP Mess Detector] has been updated from `2.2.3` to `2.4.2`.
+    The minimum supported version remains at `2.2.3`.
+
+#### Removed
+* PHPCS: The exclusion of the whole `WordPress.NamingConventions.ValidFunctionName` sniff from the YoastCS ruleset.
+
+
+### 0.1 - 2016-01-08
```

Comment:
Missing square brackets around version number.

Reply:
Actually, they are not missing :smiley_cat: 

The square brackets indicate a link and the release numbers all link to the commit summary, i.e. a compare between the previous tag and the release tag.

As version `1.0` has no previous tag, there is nothing which can be linked to.

Reply:
Do'h! Now it all makes sense /facepalm